src/movement.rs

```diff
@@ -13,10 +14,24 @@ use crate::{
     state::State, ArrivedEvent, DespawnMarker, Player, Stats,
 };
 
+// Distance after which the boundary is moved.
+//
+const LEFT_BOUNDARY_MAX_DISTANCE: f32 = 380.;
```

Comment:
Consider moving to consts.rs. I don't feel strongly about it, could see case for keeping it here where its relevant for tweaks as well.

Reply:
I've moved it, although generally speaking, I think there is value on keeping module-private constants inside the module (as they are not used anyway anywhere else).

Reply:
> I think there is value on keeping module-private constants inside the module (as they are not used anyway anywhere else).
Yeap that makes sense to me. (If you feel like putting that one back I wont object)

Reply:
No worries, I'm not fussy :laughing:

Since there is a certain overlap between the content of `consts.rs` and the yaml config, it may be worth, in the future, thinking about where to place them.

Since camera may have customization, possibly, at some point, I'll evaluate if moving `LEFT_BOUNDARY_MAX_DISTANCE` to the yaml config.

Long story short: fine for me :laughing: